docs/source/reference/structs.rst

```diff
@@ -11,6 +11,7 @@ Structs
 
 .. autoclass:: FrozenTrial
     :members:
+    :exclude-members: system_attrs, set_system_attr, trial_id
```

Comment:
`FrozenTrial` doesn't have `set_system_attr` method, so we can remove it from `exclude_members`:
```suggestion
    :exclude-members: system_attrs, trial_id
```

Reply:
Besides, I think that it's needed to remove the doc entries of `system_attrs` and `trial_id` from [here](https://github.com/pfnet/optuna/blob/a9bacebc416a2acdf6924a46ab8ff46378afc275/optuna/structs.py#L85).

Reply:
`system_attrs` seems to be public but do you want to remove it still?

Reply:
Changed as suggested.